### PR TITLE
Issue #18: CRUD Operations for DoseSeries

### PR DESCRIPTION
**Pull request by jking31cs**
Created a DoseSeries object, webservice, service, and dao, and implemented them.

### CODE REVIEW

**Comment by jking31cs** [comment]
I need to update the WIki with the http endpoint for this.  I have it as PUT "/meds/{id}/dose-series" right now.

**Comment by geoffreygraddy** [comment]
👍

File: server/src/main/java/info/doseamigos/ObjectMapperProvider.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import info.doseamigos.authusers.AuthUser;
 
 import javax.inject.Provider;
+import java.util.Date;
```
Comment (mckoon) [inline server/src/main/java/info/doseamigos/ObjectMapperProvider.java]: You only added an import to this file...
- [ ] Remove

**Comment by mckoon** [comment]
Fine, just remove your extra import.

+1

**Comment by jleopold28** [comment]
👍

**Comment by jking31cs** [comment]
Removed unused imports and fixed spelling on some java docs, merging.

**Pull request merged**